mne/viz/_3d.py

```diff
@@ -537,6 +538,8 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         .. versionadded:: 0.16
         .. versionchanged:: 1.0
            Defaults to ``'terrain'``.
+    sensor_colors: np.array
+        Array with shape (num_sensors, 3), default None
```

Comment:
in the code it looks like you're now handling input that is list of len(num_sensors), e.g., `['black', 'red', 'red', ...]`. Great!  The docstring should reflect that. Here I've also added that it should support RGBA, not just RGB (which it might already since you're doing `sensor_colors[idx, :]`). But please double-check that it works / include it in a test.

```suggestion
    sensor_colors: array-like | None
        Colors to use for the sensor glyphs. Can be list-like of color strings
        (length ``n_sensors``) or array-like of RGB(A) values (shape
        ``(n_sensors, 3)`` or ``(n_sensors, 4)``). ``None`` (the default) uses
        the default sensor colors for the :func:`~mne.viz.plot_alignment` GUI.
```